client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -522,18 +538,52 @@ class EmailProvidersComparison extends Component {
 			'email-providers-comparison__keep-main-price': ! isEligibleForFreeTrial,
 		} );
 
-		const formattedPrice = translate( '{{price/}} /mailbox /month (billed monthly)', {
+		const titanIsMonthly = titanMailProduct && isTitanMonthlyProduct( titanMailProduct );
+
+		const titanMonthlyCost = titanIsMonthly
+			? titanMailProduct?.cost ?? 0
+			: ( titanMailProduct?.cost ?? 0 ) / 12;
+
+		const monthlyPrice = formatCurrency( titanMonthlyCost, currencyCode );
+
+		const priceComponents = {
 			components: {
-				price: (
-					<span className={ formattedPriceClassName }>
-						{ formatCurrency( titanMailProduct?.cost ?? 0, currencyCode ) }
-					</span>
-				),
+				price: <span className={ formattedPriceClassName }>{ monthlyPrice }</span>,
 			},
 			comment: '{{price/}} is the formatted price, e.g. $20',
-		} );
+		};
 
-		const discount = isEligibleForFreeTrial ? translate( '3 months free' ) : null;
+		const formattedPrice = titanIsMonthly
+			? translate( '{{price/}} /mailbox /month (billed monthly)', priceComponents )
+			: translate( '{{price/}} /mailbox /month (billed annually)', priceComponents );
+
+		const discount = ! isEligibleForFreeTrial ? null : (
+			<>
+				{ translate( '3 months free' ) }
+				{ ! titanIsMonthly && (
+					<span className="email-providers-comparison__discount-with-renewal">
+						<span>
+							{ ' ' }
```

Comment:
I've suffered this a lot 😄 Somehow PhpStorm loves to add those separators when fixing the file linting automagically.

```suggestion
```